lib/graph.ts

```diff
@@ -559,6 +561,7 @@ class GraphImpl implements Graph, Graph.Transformer {
     // apply common transform
     this.removeAllIdentityNodes();
     this.removeAllDropoutNodes();
+    this.fuseConvActivationNodes();
```

Comment:
If we apply fusion here, wouldn't it fail all backends except webgl?
I'm a little surprised that browser stack CI got passed. did I 
 miss anything?

Reply:
That's my guess too. So I submit this PR and want to check how the CI goes. Will look into the result tomorrow.